### PR TITLE
Update pv_excess_control.py to improve time trigger management

### PR DESCRIPTION
**Pull request by stefan73**
Old code has problems to roubstly have a time_triggered instance of on_time() running. Often there was no instance. Especially when automation got restarted. This had to be done twice and the user was also not able to understand subsequent issues.
To fix this the multiplicty of instance copies in a global array, a local trigger variable was removed. The logics of trying to have just one on_time() method running was changed. The control code now gets only executed for the first instance in the dictionary, but all automations have a on_time() instance. This seems robust.

### CODE REVIEW

File: PV_Excess_Control/blueprints/automation/pv_excess_control.yaml [hunk 1]
```diff
@@ -1,10 +1,11 @@
 blueprint:
-  name: INVENTOCASA - PV Excess Optimizer
+  name: PV Excess Optimizer changed by Stefan73 based on Inventocasa
```
Comment (InventoCasa) [inline PV_Excess_Control/blueprints/automation/pv_excess_control.yaml:2]: There should rather be a contribution list where you can add yourself

File: PV_Excess_Control/blueprints/automation/pv_excess_control.yaml [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
   description: "### **PV EXCESS OPTIMIZER**\nAutomatically control your appliances based on excess power from your solar system <br> <br>
+  &rarr; This is a further developed version of Inventocasa's script.\n
+  &rarr; I hope changes will be pulled into the original project. In the mean time, I try to push developments here.\n
```
Comment (InventoCasa) [inline PV_Excess_Control/blueprints/automation/pv_excess_control.yaml:5]: Also cannot accept this / please change this back

File: PV_Excess_Control/README.md [hunk 3]
```diff
@@ -1,64 +1 @@
-# PV Excess Control
```
Comment (InventoCasa) [inline PV_Excess_Control/README.md:1]: Why delete the whole readme? Please readd.

File: README.md [hunk 4]
```diff
@@ -1,12 +1,11 @@
-# ha-advanced-blueprints
```
Comment (InventoCasa) [inline README.md:1]: Cannot accept these changes, since original info is deleted. Please restore.

**Comment by InventoCasa** [comment]
@stefan73 Thanks for the PR. I will review it hopefully on the upcoming weekend.
However I've already seen that you made some changes to the files which I cannot accept; I commented these changes directly at the specific code lines.

**Comment by stefan73** [comment]
I guess something did not work out with this PR. My intention was only to propose the first change for pull at this moment (
Update pv_excess_control.py to improve time trigger management). This fixes - at least for me - the unreliability of having a robustly running on:time() instance. This occurs especially if you are in the beginning starting to set up automations (and also remove them) due to the global which gets never reset and the multiplicity of references kept in the two arrays.

For the other changes I assume to be on my fork for now as a kind of alpha devel. This is e.g. why I deleted the readme because it is incomplete / inconsistent to the changes done. Or I changes the headline of the automation so that users understand that they are using the forked version. What gets merged back from there needs to be seen.

**Comment by InventoCasa** [comment]
Alright, in that case you may just update this PR (or create a new one) so that it only contains the fix for the Python script. Then I can review / merge it.

**Comment by stefan73** [comment]
Closing this - resubmitting a branch so that only the intended change can be pulled at any time.

**Pull request closed without merging**